gap/base/recognition.gd

```diff
@@ -564,7 +564,7 @@ DeclareSynonym("RecognizeGroup", RecogniseGroup);
 ##     which is created for the group <A>H</A>. Especially the component
 ##     <C>hints</C> can contain a list of additional find homomorphism
 ##     methods (described by records as in Section <Ref Sect="whataremethods"/>)
-##     which is prepended to the method database in <A>methoddb</A> before
+##     which is sorted into the method database in <A>methoddb</A> before
```

Comment:
What does 'sorted into' mean? It's not clear to me.

Reply:
Yeah I wasn't sure whether that'd be obvious. Apparently not. :) I'll clarify that. Thanks!